lib/__tests__/sandboxes.test.ts

```diff
@@ -0,0 +1,263 @@
+      mockedGetConfigAccounts.mockReturnValue([mockParentAccount]);
+
+      expect(
+        getHasSandboxesByType(
+          mockParentAccount,
+          HUBSPOT_ACCOUNT_TYPES.DEVELOPMENT_SANDBOX
+        )
+      ).toBe(false);
+    });
+  });
+
+  describe('getAvailableSyncTypes()', () => {
+    const mockParentAccount = {
+      name: 'Parent Account',
+      portalId: 123,
+      env: 'qa' as Environment,
+    };
+
+    const mockChildAccount = {
+      ...mockParentAccount,
+      portalId: 456,
+    };
+
+    beforeEach(() => {
+      jest.resetAllMocks();
+    });
+
+    it('returns available sync types when fetch is successful', async () => {
+      const mockSyncTypes = [{ name: 'type1' }, { name: 'type2' }];
+      mockedGetAccountId
+        .mockReturnValue(mockParentAccount.portalId)
+        .mockReturnValue(mockChildAccount.portalId);
+      mockedFetchTypes.mockResolvedValue({
+        data: { results: mockSyncTypes },
+      });
+
+      const result = await getAvailableSyncTypes(
+        mockParentAccount,
+        mockChildAccount
+      );
+
+      expect(result).toEqual([{ type: 'type1' }, { type: 'type2' }]);
+    });
+
+    it('throws error when sync types fetch fails', async () => {
+      mockedFetchTypes.mockResolvedValue({ data: { results: null } });
+
+      await expect(
+        getAvailableSyncTypes(mockParentAccount, mockChildAccount)
+      ).rejects.toThrow();
+    });
+  });
+
+  describe('validateSandboxUsageLimits()', () => {
+    const mockAccount = {
+      name: 'Test Account',
+      portalId: 123,
+      authType: undefined,
+      env: 'qa' as Environment,
+    };
+
+    beforeEach(() => {
+      jest.resetAllMocks();
+    });
+
+    it('validates successfully when limits are not reached', async () => {
+      mockedGetAccountId.mockReturnValue(mockAccount.portalId);
+      mockedGetSandboxUsageLimits.mockResolvedValue({
+        data: {
+          usage: { DEVELOPER: { available: 1, limit: 3 } },
+        },
+      });
+
+      await expect(
+        validateSandboxUsageLimits(
+          mockAccount,
+          HUBSPOT_ACCOUNT_TYPES.DEVELOPMENT_SANDBOX,
+          'qa'
+        )
+      ).resolves.not.toThrow();
+    });
+
+    it('throws error when development sandbox limit is reached', async () => {
+      mockedGetAccountId.mockReturnValue(mockAccount.portalId);
+      mockedGetConfigAccounts.mockReturnValue([]);
+      mockedGetSandboxUsageLimits.mockResolvedValue({
+        data: {
+          usage: { DEVELOPER: { available: 0, limit: 1 } },
+        },
+      });
+
+      await expect(
+        validateSandboxUsageLimits(
+          mockAccount,
+          HUBSPOT_ACCOUNT_TYPES.DEVELOPMENT_SANDBOX,
+          'qa'
+        )
+      ).rejects.toThrow();
+    });
+  });
+
+  describe('handleSandboxCreateError()', () => {
+    const mockEnv = 'qa' as Environment;
+    const mockName = 'Test Sandbox';
+    const mockAccountId = 123;
+
+    beforeEach(() => {
+      jest.resetAllMocks();
```

Comment:
We should have resetting the mocks globally enabled in the jest config, we should be able to remove this

Reply:
Removed in [1380930](https://github.com/HubSpot/hubspot-cli/pull/1366/commits/1380930867893625d088c394b4c0c43354980525) 👍